clients/cli/src/main/java/org/apache/gravitino/cli/ErrorMessages.java

```diff
@@ -30,6 +30,8 @@ public class ErrorMessages {
   public static final String UNKNOWN_TABLE = "Unknown table name.";
   public static final String MALFORMED_NAME = "Malformed entity name.";
   public static final String MISSING_NAME = "Missing --name option.";
+  public static final String MISSING_ENTITY =
```

Comment:
This is a bit long and seems identical to MISSING_NAME?

Reply:
yes, but Does the "Malformed entity name" message indicate that no entity is specified? I found this message a bit overwhelming when I first encountered it.

Reply:
MISSING_NAME means the name is misisng, MALFORMED_NAME means the name format is not correct i.e. it may be misisng a piece or have the wrong separators etc

Reply:
Does the "Malformed entity name" message indicate that no entity is specified? I found this message a bit overwhelming when I first encountered it.